### PR TITLE
feat(core/execution-parameters): condense parameters/artifacts and make it collapsable

### PR DESCRIPTION
**Pull request by kevinawoo**
This PR address some issues when there's lots of parameters and/or artifacts, making the pipeline page very long. On smaller screen sizes, unusable.
Included are some screenshots of what it looks like now and how it looks after applying this PR:

<details><summary>How it currently looks</summary>

<kbd></kbd>

</details>

<details open="true"><summary>Let's see a gif in action (download is kind of big, give it a sec)</summary>

<kbd>ezgif com-gif-maker (3)</kbd>

</details>

<details><summary>Collapsed View with some annotations of what's changed</summary>

<kbd></kbd>

</details>

<details><summary>Expanded View with some annotations of what's changed</summary>

<kbd></kbd>

</details>

<details><summary>Standalone View is auto expanded</summary>

<kbd></kbd>

</details>

### CODE REVIEW

File: app/scripts/modules/core/src/pipeline/executions/executions.less [hunk 1]
```diff
@@ -84,7 +84,7 @@
       }
       &.group-by-name {
         .execution-bar {
-          padding-top: 20px;
+          //padding-top: 20px;
```
Comment (kevinawoo) [inline app/scripts/modules/core/src/pipeline/executions/executions.less]: need to look at this

File: app/scripts/modules/core/src/pipeline/status/artifactList.less [hunk 2]
```diff
@@ -8,3 +10,43 @@
     margin-bottom: 4px;
   }
 }
+
+.execution-overview {
+  .resolved-artifacts {
+    font-size: 0.8em;
+    line-height: 12px;
+    margin-bottom: 10px;
+    padding-left: @execution-details-indent - 0.2em;
```
Comment (kevinawoo) [inline app/scripts/modules/core/src/pipeline/status/artifactList.less]: add comment on why there's a 0.2em

File: app/scripts/modules/core/src/pipeline/status/executionParameters.less [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@import (reference) '~core/pipeline/executions/execution/execution.less';
+
+.execution-overview {
+  .execution-parameters {
+    font-size: 0.8em;
+    line-height: 12px;
+    overflow: scroll;
+    margin-bottom: 10px;
+    padding-left: @execution-details-indent - 0.2em;
```
Comment (kevinawoo) [inline app/scripts/modules/core/src/pipeline/status/executionParameters.less]: add comment on why there's a 0.2em

File: app/scripts/modules/core/src/pipeline/executions/executions.less (hunk 1 above)
Comment (kevinawoo) [reply app/scripts/modules/core/src/pipeline/executions/executions.less]: does it look weird?

File: app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx [hunk 4]
```diff
@@ -379,7 +415,45 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
...
+              collapseParamsArtifactsAfter - resolvedExpectedArtifacts.length,
+            )}
+          />
+
+          {SETTINGS.feature.artifacts && (
+            <ResolvedArtifactList
+              artifacts={artifacts}
+              resolvedExpectedArtifacts={resolvedExpectedArtifacts}
+              showingExpandedArtifacts={showingParams}
+              columnFormatAfter={Math.min(collapseParamsArtifactsAfter, collapseParamsArtifactsAfter - parametersCount)}
```
Comment (ncknt) [inline app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx]: Remove the Math.min() -> `collapseParamsArtifactsAfter - parametersCount`

**Pull request closed without merging**